### PR TITLE
Draco js module build instructions

### PR DESCRIPTION
**Pull request by ggetz**
Added instructions for updating the draco module, following PR with update to draco build options: https://github.com/google/draco/pull/405

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Signed CLA is on file.

@ggetz, thanks for the pull request! Maintainers, we have a signed CLA from @ggetz, so you can review this at any time.

:warning: I noticed that [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md) has not been updated. If this change updates the public API in any way, fixes a bug, or makes any non-trivial update, please add a bullet point to `CHANGES.md` and comment on this pull request so we know it was updated. For more info, see the [Pull Request Guidelines]( https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines).

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

**Comment by ggetz** [comment]
Thanks @lilleyse, updated! 

Added a line preferring Linux or the subsystem, but left previous instructions as an optional backup.

**Comment by lilleyse** [comment]
Looks good. Does anyone else have feedback or want to try this out?

**Comment by mramato** [comment]
Looks good to me.  Sounds like building is trivial if you're already familiar with cmake/unix dev tools.  Thanks @ggetz!

File: Documentation/Contributors/DracoModuleManagement/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## Updating the Draco JavaScript Decoder Module
+
+We use a custom build of the [Draco](https://github.com/google/draco) decoder JavaScript module to allow for IE11 compatibility.
+
+
+1. Download and install a make tool.
+    * Preferably, use Linux or [the Linux Subsystem for Windows 10](https://docs.microsoft.com/en-us/windows/wsl/install-win10) and `make` to build.
+    * Optionally, [MSYS2](http://www.msys2.org/). Follow installation instructions and add the path to MSYS2's usr/bin to your `PATH`. For example, if the path is `C:\msys64\usr\bin`, run `PATH=%PATH%;C:\msys64\usr\bin`.
+2. Clone or download the [Draco release](https://github.com/google/draco/releases).
```
Comment (mramato) [inline Documentation/Contributors/DracoModuleManagement/README.md]: Actually, one comment.  If you clone Draco from git, you'll get master and not an official release, right?  We should probably always include an official release in Cesium

**Comment by ggetz** [comment]
Thanks @mramato. Meant to imply that, but it wasn't particularly clear so I made it more explicit.

**Comment by lilleyse** [comment]
The updated wording looks good to me.

**Pull request merged**